python/ribasim/tests/test_io.py

```diff
@@ -62,7 +62,7 @@ def test_basic_transient(basic_transient, tmp_path):
     time = model_loaded.basin.time
     assert model_orig.basin.time.df.time[0] == time.df.time[0]
     __assert_equal(model_orig.basin.time.df, time.df)
-    assert time.df.shape == (1468, 9)
+    assert time.df.shape == (1468, 8)
```

Comment:
Is this because the "remarks" string column (needed as a hack for pandera) is gone? There are a few other occurances of "remarks" in the test code that would be good to update.

Reply:
thanks, done